test/helpers/clusterlogging.go

```diff
@@ -28,7 +28,7 @@ func NewClusterLogging(componentTypes ...LogComponentType) *cl.ClusterLogging {
 			Name:      ClusterLoggingName,
 			Namespace: OpenshiftLoggingNS,
 			Annotations: map[string]string{
-				k8shandler.ForwardingAnnotation: "enabled",
+				k8shandler.PreviewForwardingAnnotation: "enabled",
```

Comment:
is this annotation still required if we're going GA?

Reply:
No, needs cleaning up.

Reply:
No, will remove.